contentcuration/contentcuration/viewsets/base.py

```diff
@@ -195,13 +195,15 @@ def update(self, queryset, all_validated_data):
         m2m_fields_by_id = {}
 
         for obj in objects_to_update:
-            obj_id = getattr(obj, id_attr)
+            # Coerce to string as some ids are of the UUID class
+            obj_id = str(getattr(obj, id_attr))
```

Comment:
I think this definitely makes me want to not handle M2M fields via fields on the respective models, and split it out separately (much like I did for the user channel edit and viewing).